eventlet/wsgi.py

```diff
@@ -748,6 +748,9 @@ def get_environ(self):
 
         env['headers_raw'] = headers_raw = tuple((k, v.strip(' \t\n\r')) for k, v in headers)
         for k, v in headers_raw:
+            if "_" in k:
```

Comment:
I'd suggest either to internalize the treatment of this condition and of the following replacement (from line 751 to line  754) into a dedicated function with a an explicit naming (e.g `formalize_key_naming`) and with an explicit docstring explain the purpose of doing this, or to add a comment just before the condition you added here.

I think that without more explanations than that, at the implementation level, many people could wonders why this condition exist.